cves/CVE-2017-14058.yml

```diff
@@ -64,7 +64,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: a read function does not restrict reload attempts for a bad list, so a denial of service via infite loop can occur
```

Comment:
Like the first one this needs a lot more context and detail. Going to end my review here - I think the feedback on the first one gives you a lot of direction.